### PR TITLE
Create ListCheckbox kit

### PR DESCRIPTION
**Pull request by andrew-k9**
#### Screens

Screen Shot 2020-08-24 at 1 29 12 PM

#### Description

Student implementation of [184](https://github.com/powerhome/playbook/issues/184)

### CODE REVIEW

File: app/pb_kits/playbook/pb_list_checkbox/_list_checkbox.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
+
+
+/* @flow */
+
+import React from 'react'
```
Comment (kellyeryan) [inline app/pb_kits/playbook/pb_list_checkbox/_list_checkbox.jsx:5]: Make sure to import: 

import { buildAriaProps, buildCss, buildDataProps } from '../utilities/props'
import classnames from 'classnames'
import { globalProps } from '../utilities/globalProps.js'

File: app/pb_kits/playbook/pb_list_checkbox/_list_checkbox.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import { List } from '../'
+
+type ListCheckboxProps = {
+  aria?: object,
+  children: array<Node> | Node,
+  className?: string,
+  data?: object,
+  id?: string,
+}
+
```
Comment (kellyeryan) [inline app/pb_kits/playbook/pb_list_checkbox/_list_checkbox.jsx:16]: Check out the example in `checkbox.jsx`

File: app/pb_kits/playbook/packs/examples.js [hunk 3]
```diff
@@ -173,3 +173,5 @@ WebpackerReact.setup({
   ...WeekdayStacked,
 })
 
+import * as ListCheckbox from 'pb_list_checkbox/docs'
+WebpackerReact.setup(ListCheckbox)
```
Comment (garettarrowood) [inline app/pb_kits/playbook/packs/examples.js:177]: Move this to be imported and set the same way all the other kits are getting imported and setup.

**Comment by garettarrowood** [comment]
This kit is to be merged with the ListRadio kit which is further along. @andrew-k9 - You will join Jo in finishing and expanding her PR to encapsulate these sets of logic.

**Pull request closed without merging**